core/src/main/java/io/questdb/griffin/ExpressionParser.java

```diff
@@ -35,7 +35,7 @@
     private static final IntHashSet nonLiteralBranches = new IntHashSet();
     private static final int BRANCH_NONE = 0;
     private static final int BRANCH_COMMA = 1;
-    private static final int BRANCH_LEFT_BRACE = 2;
+    private static final int BRANCH_LEFT_BRACE = 2; // brace -> parenthesis
```

Comment:
Since this is `private static`, why not simply fix this and rename to BRANCH_LEFT_PARENTHESIS and BRANCH_RIGHT_PARENTHESIS throughout?

Reply:
done it!